pkg/resource/managed_reconciler.go

```diff
@@ -43,7 +43,12 @@ const (
 
 // Error strings.
 const (
-	errGetManaged = "cannot get managed resource"
+	errGetManaged       = "cannot get managed resource"
+	errReconcileConnect = "Connect failed"
+	errReconcileObserve = "Observe failed"
+	errReconcileCreate  = "Create failed"
+	errReconcileUpdate  = "Update failed"
+	errReconcileDelete  = "Delete failed"
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Do you think this satisfies the above rule? If we assume `Connect` etc are proper nouns, i.e. names of the method that failed, it does. If we think of them as generic actions it probably does not.

Reply:
I would tend to think of them as generic actions, will update :)